server/plugin/api_test.go

```diff
@@ -1041,85 +1155,74 @@ func TestHandleEndPoll(t *testing.T) {
 				store.PollStore.On("Get", testutils.GetPollID()).Return(nil, &model.AppError{})
 				return store
 			},
-			Request:            &model.PostActionIntegrationRequest{UserId: "userID1", ChannelId: "channelID1", PostId: "postID1"},
+			Request:            &model.SubmitDialogRequest{UserId: "userID1", CallbackId: "postID1", TeamId: "teamID1"},
```

Comment:
I want `ExpectedResponse: nil,` in this test case like others.

Reply:
You've got really good eyes :+1: